tailscale/resource_dns_configuration.go

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) David Bond, Tailscale Inc, & Contributors
+// SPDX-License-Identifier: MIT
+
+package tailscale
+
+import (
+	"context"
+	"slices"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"tailscale.com/client/tailscale/v2"
+)
+
+func resourceDNSConfiguration() *schema.Resource {
+	return &schema.Resource{
+		Description:   "WARNING: this is currently alpha functionality, proceed with caution! The dns_configuration resource allows you to manage the complete DNS configuration for your Tailscale network. See https://tailscale.com/kb/1054/dns for more information.",
+		ReadContext:   resourceDNSConfigurationRead,
+		CreateContext: resourceDNSConfigurationCreate,
+		UpdateContext: resourceDNSConfigurationUpdate,
+		DeleteContext: resourceDNSConfigurationDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"nameservers": {
+				Description: "Set the nameservers used by devices on your network to resolve DNS queries.",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"address": {
+							Description: "The nameserver's IPv4 or IPv6 address",
+							Type:        schema.TypeString,
+							Required:    true,
+						},
+						"use_with_exit_node": {
+							Description: "This name server will continue to be used when an exit node is selected (requires Tailscale v1.88 or later).",
```

Comment:
Nit:
```suggestion
							Description: "This name server will continue to be used when an exit node is selected (requires Tailscale v1.88.1 or later).",
```

Reply:
The UI says 1.88, is it wrong?

<img width="489" height="87" alt="image" src="https://github.com/user-attachments/assets/2b4a84c4-32c6-4435-9e45-dd9545eabcc5" />

Reply:
I'll leave it as-is for consistency with the UI.